lib/gltfStatistics.js

```diff
@@ -0,0 +1,202 @@
+'use strict';
+var Cesium = require('cesium');
+
+var defaultValue = Cesium.defaultValue;
+var defined = Cesium.defined;
+
+module.exports = {
+    gltfStatistics,
+    getDrawCallStatistics,
+    printStats
+}
+
+function getBuffersSize(buffers) {
+    var size = 0;
+    for (var name in buffers) {
```

Comment:
Instead of `name`, use `id` which is the proper glTF terminology. Same comment for the other loops.